woocommerce/woocommerce
# update external relative links to full URLs

### Changes proposed in this Pull Request:

Some of the markdown files in the `docs` folder include relative links to markdown files that are in the repo but outside the docs folder. These markdown files are not included in the manifest and the ingestion plugin cannot replace the relative links.

This PR updates those relative links to be full URLs.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In your docs dev install, edit the `Naming Conventions` post
2. Click on the `README for /src` link
3. Note that the link url is `../../plugins/woocommerce/src/README.md`
4. Review the link replacements in this PR to verify that they are all full valid URLs (due to not being able to ingest these updated docs until they are merged)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
